zinc/src/sbt-test/source-dependencies/issue-810/build.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
the name of the folder `issue-810` is okay?

Reply:
I prefer descriptive names. Something like `subproject-api-update`?